android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/main/MainScreen.kt

```diff
@@ -0,0 +1,21 @@
+package co.nimblehq.ic.kmm.suv.android.ui.screens.main
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import co.nimblehq.ic.kmm.suv.Greeting
+
+@Composable
+fun MainScreen() {
+    Box(modifier = Modifier.fillMaxSize()) {
+        Text(
+            text = Greeting().greeting(),
+            color = Color.Black,
+            modifier = Modifier.align(Alignment.Center)
```

Comment:
Same here. `modifier` should be on top 😄

Reply:
Replied here: https://github.com/suho/kmm-ic/pull/79#discussion_r976041121